chromium_src/chrome/browser/ui/views/download/bubble/download_toolbar_button_view.h

```diff
@@ -0,0 +1,36 @@
+/* Copyright (c) 2023 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_CHROMIUM_SRC_CHROME_BROWSER_UI_VIEWS_DOWNLOAD_BUBBLE_DOWNLOAD_TOOLBAR_BUTTON_VIEW_H_
+#define BRAVE_CHROMIUM_SRC_CHROME_BROWSER_UI_VIEWS_DOWNLOAD_BUBBLE_DOWNLOAD_TOOLBAR_BUTTON_VIEW_H_
+
+#define DownloadToolbarButtonView DownloadToolbarButtonViewChromium
+#define GetIconColor                                      \
+  UnUsed() {                                              \
+    return SK_ColorTRANSPARENT;                           \
+  }                                                       \
+                                                          \
+ protected:                                               \
+  DownloadDisplayController* display_controller() const { \
```

Comment:
Added to call this from the const method.

Reply:
nit: returning mutable from `const` is not great, so another option here is to implement this as `DownloadDisplayController::IconInfo GetIconInfo() const { return controller_->GetIconInfo(); }`.

Reply:
I think the return type should be `const DownloadDisplayController*` based on https://chromium.googlesource.com/chromium/src/+/HEAD/styleguide/c++/const.md - The returned member could be changed so this getter doesn't seem to be logically const.

---

But changing return type like so would make us use const_cast<>, so I think we can remove this getter here and just use const_cast<> from the callsite.

Reply:
I just saw @goodov right after I wrote the comment above. +1 for @goodov 's suggestion.

Reply:
Thanks both! https://chromium.googlesource.com/chromium/src/+/HEAD/styleguide/c++/const.md is nice doc :)